eclair-core/src/main/scala/fr/acinq/eclair/router/RouteCalculation.scala

```diff
@@ -65,14 +66,17 @@ object RouteCalculation {
             case edges if edges.nonEmpty && edges.forall(_.nonEmpty) =>
               // select the largest edge (using balance when available, otherwise capacity).
               val selectedEdges = edges.map(es => es.maxBy(e => e.balance_opt.getOrElse(e.capacity.toMilliSatoshi)))
-              val hops = selectedEdges.map(e => ChannelHop(getEdgeRelayScid(d, localNodeId, e), e.desc.a, e.desc.b, e.params))
+              val hops = selectedEdges.map(e => e.params match {
+                case params: ChannelRelayParams => ChannelHop(getEdgeRelayScid(d, localNodeId, e), e.desc.a, e.desc.b, params)
+                case params: HopRelayParams.FromBlindedRoute => BlindedHop(params.dummyId, params.route, params.paymentInfo)
```

Comment:
Is it actually possible to get a blinded edge here?

Reply:
At first I thought no, but actually we can if we add them to `extraEdges`, which could potentially make sense?